src/frontend/pages/report-history.vue

```diff
@@ -11,25 +11,19 @@
     </v-row>
     <v-row v-else>
       <v-col v-for="report in reports" :id="report.id" :key="report.id" cols="12" class="pa-2">
-        <Success v-if="!report.data.error && report.data.done" :report="report" />
-        <In-progress v-if="!report.data.error && !report.data.done" :report="report" />
-        <Error v-if="report.data.error" :report="report" />
+        <ReportCard :report="report" :status="getStatusOfReport(report)" />
```

Comment:
Please use computed properties in the `ReportCard` component instead of populating `status` with a method

Reply:
i forget to commit